docker/Dockerfile.consensus-worker

```
@@ -23,9 +23,9 @@ RUN apt-get update -y \
     && rm -rf /var/lib/apt/lists/*
 
 # Install Python dependencies
-ADD backend/protocol_rpc/requirements.txt backend/protocol_rpc/requirements.txt
+ADD backend/requirements.txt backend/requirements.txt
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)